modules/oxide/src/less/theme/components/comments/comment.less

```diff
@@ -204,6 +205,7 @@
 
   .tox-conversations {
     margin: @pad-sm;
```

Comment:
I don't want to remove overflow from the scroll element or it changes the design. I'm pretty sure the scroll element becomes 0px already when the textarea gets too tall - but it's hard to tell because then _nothing_ can be scrolled.